packages/workbox-core/src/controllers/WorkboxCore.js

```diff
@@ -0,0 +1,20 @@
+import assert from '../utils/assert';
+import logHelper from '../utils/logHelper';
+
+export default class WorkboxCore {
+  get INTERNAL() {
+    if (this._internal) {
+      return this._internal;
+    }
+
+    this._internal = {
+      logHelper,
+    };
+
+    if (process.env.NODE_ENV !== 'production') {
+      this._internal.assert = assert;
+    }
```

Comment:
Yes, if you look at the tests I believe there is a check where depending on the node_env var will determine if the test expects it to exist or not